codespell_lib/data/dictionary.txt

```diff
@@ -38553,12 +38568,19 @@ valies->values
 valif->valid
 valitdity->validity
 valkues->values
+vallay->valet
```

Comment:
Perhaps `valley` (and plural forms) too?

Reply:
Added.